e2e/e2etests/test_precompiles.go

```diff
@@ -0,0 +1,94 @@
+package e2etests
+
+import (
+	"math/big"
+	"strings"
+
+	"cosmossdk.io/math"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+
+	"github.com/zeta-chain/node/e2e/contracts/istaking"
+	"github.com/zeta-chain/node/e2e/runner"
+)
+
+// source: https://github.com/cosmos/evm/blob/main/precompiles/staking/StakingI.sol
+const stakingAddr = "0x0000000000000000000000000000000000000800"
```

Comment:
consider making this part of config or adding into the the runner directly

Reply:
I would keep as is right now as it is a niche test.
Might change in the future.